frontend/src/pages/_app.js

```diff
@@ -21,19 +21,29 @@ export default function App({ Component, pageProps }) {
 
   const getLayout = (page) => {
     if (pageIsForAuth || pageIsIndex) {
-      return <BlankLayout children={page} />;
+      return (
+        <>
+          <BlankLayout children={page} />
+          <Toaster position="top-center" reverseOrder={false} />
+        </>
+      );
     }
 
     if (pageIsForSettings) {
       return (
-        <Layout>
+        <Layout children={page}>
```

Comment:
I don't think you need this.

Imo it should be a fragment with the nested SettingsLayout serving the actual children.

Please make this change and test it out once.